.github/workflows/prerelease.yml

```diff
@@ -107,15 +104,14 @@ jobs:
 
   test_ubuntu_clang:
     name: Ubuntu (Clang 16)
+    if: github.ref == 'refs/heads/main'
```

Comment:
What is the logic here?

Reply:
Avoiding things not running on a test branch